### PR TITLE
Clean up duplication in settings.

### PR DESCRIPTION
**Pull request by prescod**

### CODE REVIEW

File: config/settings/base.py [hunk 1]
```diff
@@ -274,3 +275,59 @@
...
+
+SALESFORCE_DOCSET_SOBJECT = env("SALESFORCE_DOCSET_SOBJECT", default="Hub_Product_Description__c")
+SALESFORCE_DOCSET_ID_FIELD = env("SALESFORCE_DOCSET_ID_FIELD", default="EasyDITA_UUID__c")
+SALESFORCE_DOCSET_STATUS_FIELD = env("SALESFORCE_DOCSET_STATUS_FIELD", default="Status__c")
+SALESFORCE_DOCSET_STATUS_INACTIVE = env("SALESFORCE_DOCSET_STATUS_FIELD", default="Inactive")
+SALESFORCE_DOCSET_INDEX_REFERENCE_FIELD = env("SALESFORCE_DOCSET_INDEX_REFERENCE_FIELD", default="Index_Article_Id__c")
+SALESFORCE_DOCSET_RELATION_FIELD = SALESFORCE_DOCSET_SOBJECT
+
+# this will slow things down and should only be used for testing
+CACHE_VALIDATION_MODE = env("CACHE_VALIDATION_MODE", default=False)
```
Comment (davisagli) [inline config/settings/base.py]: I still see this one in the other files.

File: config/settings/integration_tests.py [hunk 2]
```diff
@@ -73,56 +66,9 @@
...
-SALESFORCE_ARTICLE_TYPE = env("SALESFORCE_ARTICLE_TYPE")
-SALESFORCE_ARTICLE_BODY_FIELD = env("SALESFORCE_ARTICLE_BODY_FIELD")
-SALESFORCE_ARTICLE_URL_PATH_PREFIX = env(
-    "SALESFORCE_ARTICLE_URL_PATH_PREFIX", default="/articles/Resource/"
-)
-SALESFORCE_ARTICLE_TEXT_INDEX_FIELD = env(
-    "SALESFORCE_ARTICLE_TEXT_INDEX_FIELD", default=False
-)
-SALESFORCE_ARTICLE_LINK_LIMIT = env("SALESFORCE_ARTICLE_LINK_LIMIT", default=100)
-SALESFORCE_ARTICLE_PREVIEW_URL_PATH_PREFIX = '/preview'
```
Comment (davisagli) [inline config/settings/integration_tests.py]: Please confirm it's okay for this one to change

File: env.example [hunk 3]
```diff
@@ -21,12 +17,6 @@ EASYDITA_PASSWORD='REPLACEME!!!'
 EASYDITA_USERNAME=REPLACEME!!!
 
 SALESFORCE_API_VERSION=41.0
-SALESFORCE_ARTICLE_AUTHOR_FIELD=External_Username__c
-SALESFORCE_ARTICLE_AUTHOR_OVERRIDE_FIELD=Author_User_Id_Override__c
-SALESFORCE_ARTICLE_BODY_FIELD=Raw_HTML__c
-SALESFORCE_ARTICLE_TEXT_INDEX_FIELD=Article_Body__c
-SALESFORCE_ARTICLE_TYPE=Resource__kav
-SALESFORCE_ARTICLE_URL_PATH_PREFIX=/s/article/
```
Comment (davisagli) [inline env.example]: Please confirm it's okay for this one to change

File: config/settings/base.py (hunk 1 above)
Comment (prescod) [reply config/settings/base.py]: 873d9bb

File: config/settings/integration_tests.py (hunk 2 above)
Comment (prescod) [reply config/settings/integration_tests.py]: It was safe for it to change but there was no reason for it to do so.

873d9bb

File: env.example (hunk 3 above)
Comment (prescod) [reply env.example]: Good observation. The config files started out inconsistent on this URL. I've fixed it up in

59eeba5

**Pull request merged**